build.gradle

```diff
@@ -14,9 +17,61 @@ repositories {
 
 dependencies {
     api("org.openmicroscopy:omero-blitz:5.5.0-m7")
+    api("ome:formats-api:5.9.2") {
```

Comment:
excluding Bf in general means insight will have to inject formats-bsd i.e. insight will need to know the version set in omero-model!

Reply:
Need more than that on client side. Currently not possible to import data

Reply:
This line is currently being applied _after_ `foreach-set-deps` so that the end result is not getting  6.1.0-m1:

```
/opt/omero-build $ git submodule foreach 'git grep -E "formats-(gpl|api|bsd)" snoopy/south_merge_trigger || echo none'
Entering 'omero-blitz'
none
Entering 'omero-common'
none
Entering 'omero-java-gateway'
snoopy/south_merge_trigger:build.gradle:    api("ome:formats-api:5.9.2") {
Entering 'omero-model'
snoopy/south_merge_trigger:build.gradle:    api("ome:formats-gpl:6.1.0-m1")
```

Reply:
the main question for me is: do we want the BF version to "leak" into a client importing data e.g. insight
As it stands, such clients will have to re-import BF and other modules
I did some simple tests (import tiff):
This is the change I had to make so we do not inject BF into insight. I am using BF 6.1.0-m1:
```
api("ome:formats-api:6.1.0-m1")
api("ome:formats-gpl:6.1.0-m1")  #otherwise error in the insight log reader not found
api("ome:formats-bsd:6.1.0-m1") #otherwise error in the insight log reader not found
```
comment out few excludes:
```
//exclude group: 'com.esotericsoftware.kryo'
//exclude group: 'joda-time'
//exclude group: 'ome', module: 'turbojpeg'
//exclude group: 'org.ini4j'
//exclude group: 'org.scijava'
```
i assume that ``exclude group: 'edu.ucar'`` will have to be un-excluded

This requires discussion

Reply:
BF is in a way a server and client dependency (at least for some clients) insight and gateway (cf. new work) will need it